test/cpp/test_tensor.cpp

```diff
@@ -205,13 +206,14 @@ TEST_F(TensorTest, TestViewMod) {
   output.add_(one, 1.0);
   input.add_(one, 1.0);
   ForEachDevice([&](const Device& device) {
-    at::Tensor xinput =
-        at::zeros({32, 20, 4, 4}, at::TensorOptions(at::kFloat));
-    XLATensor dev_input = XLATensor::Create(xinput, device);
-    XLATensor dev_one = XLATensor::Create(one, device);
-    XLATensor dev_output = XLATensor::view(dev_input, {-1, 320});
-    XLATensor::add_(dev_output, dev_one, 1.0);
-    XLATensor::add_(dev_input, dev_one, 1.0);
+    at::Tensor dev_input =
```

Comment:
I am fine with all the changes you made in this file but I am wondering if it is just a clean up or the old way of creating XLATensors and call `XLATensor::add_` won't work?

Reply:
The old way doesn't work because I deleted the `XLATensor::add_` method. For consistency, I thought I'd just delete all of the unnecessary inplace methods that live directly on `XLATensor`, but that required updating these tests - they now use the codegen'd `add_` kernels by invoking the pytorch C++ API, which eventually dispatches to the codegen'd XLA kernel.